src/ripple/proto/org/xrpl/rpc/v1/transaction.proto

```diff
@@ -55,8 +55,9 @@ message Transaction
 
         SignerListSet signer_list_set = 28;
 
-        TrustSet trust_set = 29;
+        TicketCreate ticket_create = 30;
```

Comment:
I know this was done intentionally, but I vote to move this to the end. I'd rather list them in id order and break alphabetical order rather than the other way around. (No change needed, just stating a preference).

Reply:
Thanks @seelabs.  @cjcobb23, do you have a preference for this ordering?  Since these proto definitions were primarily your work I'd like to do it the way you prefer.

Reply:
I prefer alphabetical order here. It's just easier to read. The ids are only really used internally and don't really mean anything to humans.